autotest.pm

```diff
@@ -173,9 +173,13 @@ sub run_all {
         warn $@;
         $died = 1;    # test execution died
     }
-    bmwqemu::save_vars();
-    myjsonrpc::send_json($isotovideo, {cmd => 'tests_done', died => $died, completed => $completed});
-    close $isotovideo;
+    eval {
+        # When the disk runs out of space there's nothing else we can do. We just can't allow to jump the exit
+        bmwqemu::save_vars();
+        myjsonrpc::send_json($isotovideo, {cmd => 'tests_done', died => $died, completed => $completed});
+        close $isotovideo;
+    };
```

Comment:
In the logs, there were some exceptions in this step. I decided to put it in an eval to avoid, the propagation of the exception skipping the _exit (this is a fork process we don't want to leave kids loose).